js/templates/modals/purchase/receipt.html

```diff
@@ -4,12 +4,12 @@
   const listingCurrency = ob.listing.metadata.pricingCurrency;
   const viewingCurrency = ob.displayCurrency;
   const isBTC = viewingCurrency === 'BTC';
-  ob.items.forEach((item, i) => {
+  ob.prices.forEach((priceObj, i) => {
```

Comment:
This was shifted to the priceObj, since it had the same data in a more concise form, and is now always up to date.